lib/swagger/swaggerTypes.ts

```diff
@@ -228,6 +228,7 @@ export interface Schema extends BaseSchema {
   oneOf?: Schema[];
   additionalProperties?: boolean | Schema;
   properties?: { [propertyName: string]: Schema };
+  patternProperties?: { [propertyPattern: string]: Schema };
```

Comment:
What is this used for?

Reply:
It's part of JSON schema, to specify schema for certain pattern properties in Json Schema. https://json-schema.org/understanding-json-schema/reference/object.html#pattern-properties